packages/react/src/Field/Field.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+export type FieldProps = {
+  /** Whether the field has an input with a validation error */
+  hasError?: boolean
```

Comment:
Reusing the word `invalid` here prevents introducing a different term that refers to the same concept.